.github/workflows/ci.yml

```diff
@@ -0,0 +1,117 @@
+name: Continuous Integration testing
+on:
+  push:
+    branches:
+      - meson
+  pull_request:
+    branches:
+      - meson
```

Comment:
This should also be changed to `master` (see comment above)

Reply:
Thank you for pointing this out. I have committed a modified .github/workflows/ci.yml so the tests will be done on master branch.